acceptance-tests/lib/env.js

```diff
@@ -52,39 +50,47 @@ const envOverrides = getTruthyValuesOnly({
 });
 
 const getTestConfig = () => {
-  // Test-specific Overrides > Command-line Args > Env vars
-  const config = Object.assign(
-    {},
-    envOverrides,
-    localOverrides,
-    localTestOverrides
-  );
+  // Command-line Args > Env vars
+  const config = Object.assign({}, envOverrides, argsOverrides);
 
-  if (!config.portalId)
+  if (!config.portalId) {
     throw new Error(
-      'accountId must be defined.  Either set the ACCOUNT_ID environment variable, or use the --accountId flag to pass it in.'
+      'accountId must be defined. Either set the ACCOUNT_ID environment variable or use the --accountId flag to pass it in.'
     );
+  }
 
-  if (!config.cliPath)
-    throw new Error(
-      'cliPath must be defined.  Either set the CLI_PATH environment variable or use the --cliPath flag to pass it in.'
-    );
+  if (!config.cliPath) {
```

Comment:
This makes it so there's one less argument that you need to pass to the script when running it